test/e2e/txmgr/txSender.go

```diff
@@ -0,0 +1,146 @@
+package txmgr
+
+import (
+	"context"
+	"crypto/ecdsa"
+	"math/big"
+	"strings"
+
+	"github.com/omni-network/omni/contracts/bindings"
+	"github.com/omni-network/omni/lib/errors"
+	"github.com/omni-network/omni/lib/log"
+	"github.com/omni-network/omni/lib/netconf"
+	"github.com/omni-network/omni/lib/txmgr"
+	t "github.com/omni-network/omni/lib/txmgr"
+	"github.com/omni-network/omni/lib/xchain"
+
+	"github.com/ethereum/go-ethereum/accounts/abi"
+	"github.com/ethereum/go-ethereum/common"
+	"github.com/ethereum/go-ethereum/ethclient"
+)
+
+type TxSender struct {
```

Comment:
instead of requiring many instances of this, one for each chain, lets rather refactor this to support sending to all chains. So all `SendX` methods should take a `srcChainID`

Reply:
wouldn't I need many instances for this though if I wanted to take in srcChainID @Lazar955 ?